src/components/components/Footer/Footer.scss

```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```

Comment:
you can remove this import, as the font is already imported in the global stylesheet